src/style.css

```diff
@@ -1 +1,94 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
```

Comment:
this is default style, you don't need it

Reply:
done